src/databricks/labs/ucx/hive_metastore/locations.py

```diff
@@ -72,7 +72,7 @@ def _parse_location(cls, location: str | None) -> list[str]:
         parse_result = cls._parse_url(location.rstrip("/"))
         if not parse_result:
             return []
-        parts = [parse_result.scheme, parse_result.netloc]
+        parts = [parse_result.scheme.replace("s3a", "s3"), parse_result.netloc]
```

Comment:
Maybe move this to a separate "clean" method as well and document why this is needed

Reply:
Done